### PR TITLE
Data fetching script

### PR DESCRIPTION
**Pull request by jimdickinson**

### CODE REVIEW

File: stardog_fetcher.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import time
+
+context_str = """kube1
+kube2
+kube3"""
+
+contexts = context_str.split('\n')
+
+for kubecontext in contexts:
+    print("working on {}".format(kubecontext))
```
Comment (EricBorczuk) [inline stardog_fetcher.py:12]: wow so python 2 compliant, so wow

File: stardog_fetcher.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    os.mkdir("/tmp/{}".format(kubecontext))
+
+    filename = "/tmp/{}/pods.json".format(kubecontext)
+    print("writing to {}".format(filename))
+    subprocess.run(args=[
+        "bash", "-c",
+        "kubectl --context={} get pods -A -o json > {}".format(kubecontext, filename)
+        ])
+    time.sleep(2)
```
Comment (EricBorczuk) [inline stardog_fetcher.py:21]: what's the purpose of sleeping for two seconds here? Does subprocess.run not run until completion?

**Review by EricBorczuk — APPROVED** [review]

File: stardog_fetcher.py (hunk 1 above)
Comment (jimdickinson) [reply stardog_fetcher.py:12]: oh yeah this is not the cool way to make strings anymore? D:

this didn't exist when Python 2.7 came out, I think

File: stardog_fetcher.py (hunk 2 above)
Comment (jimdickinson) [reply stardog_fetcher.py:21]: I would not like to totally beat down kube

**Pull request merged**